python-mini-projects/projects/All_links_from_given_webpage/get_links.py

```diff
@@ -0,0 +1,17 @@
+import requests as rq
+from bs4 import BeautifulSoup
+
+url = input("Enter Link: ")
+if ("https" or "http") in url:
+    data = rq.get(url)
+else:
+    data = rq.get("https://" + url)
+soup = BeautifulSoup(data.text, "html.parser")
+links = []
+for link in soup.find_all("a"):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)